### DISCUSSION TITLE
Meta Title is Being Used as URL Instead of Slug

### DISCUSSION THREAD

**Discussion by sudeep2003** [bagisto/bagisto#10158] [Q&A] [answered]
In my Bagisto setup, the meta title is being used to generate URLs, but it should be the slug generating the URLs for SEO-friendly links. The meta title should only be used for SEO purposes in the page's `<title>` tag, not for URL generation.

Please provide assistance on how to resolve this issue or confirm if this is a bug.

Look at the provided [screenshot](https://github.com/sudeep2003/screenshot/blob/main/abcd/Screenshot%20(542).png).
![Screenshot (542)](https://github.com/user-attachments/assets/3dbe4cc2-0ad7-4fae-b9c3-c054e5fae2ec)

**Comment by suraj-webkul** [comment, marked as answer]
Hello @sudeep2003,

The SEO component is currently just for demonstration purposes. The actual workflow is functioning as expected and works fine.

Please let me know if you need further clarification or assistance!

Best regards,  
Suraj

**Comment by amit-webkul** [comment]
This is a duplicate of the [discussion](https://github.com/bagisto/bagisto/discussions/10156)
